repositories.md

```diff
@@ -5,25 +5,23 @@ css_id: repositories
 
 # Setting up Uptane repositories
 
-This page outlines recommended procedures for the one-time operations that an OEM and its suppliers SHOULD perform when they set up Uptane for the first time. In particular, they SHOULD correctly configure the director and image repositories, as well as the time server, so that the impact of a repository / server compromise is limited to as few ECUs as possible.
+This page outlines recommended procedures for the one-time operations that an OEM and its suppliers SHOULD perform when they set up Uptane for the first time. In particular, they SHOULD correctly configure the Director and Image repositories, and, if used, the Time Server, so that the impact of a repository/server compromise is limited to as few ECUs as possible.
 
 ## Secure Source of Time
 
-Without access to a secure source of time, ECUs may be prevented from receiving the most recent updates. If the ECU's time is too high, the ECU will detect that the current valid metadata is expired and will be unable to perform an update. If the ECU's time is too low, an attacker can freeze or replay old metadata to the ECU.  (ECUs in Uptane will not accept an earlier time than what has been seen before signed with the same key.)
+Without access to a secure source of time, ECUs may be prevented from receiving the most recent updates. If the ECU's time is too fast, it will detect that the current valid metadata is expired and thus be unable to perform an update. If the ECU's time is too slow, an attacker can freeze or replay old metadata to the ECU.  (ECUs in Uptane will not accept an earlier time than what has been seen before signed with the same key.)
```

Comment:
```suggestion
Without access to a secure source of time, ECUs may be prevented from receiving the most recent updates. If the ECU's time is ahead too much, it will detect that the current valid metadata is expired and thus be unable to perform an update. If the ECU's time is too far behind, an attacker can freeze or replay old metadata to the ECU.  (ECUs in Uptane will not accept an earlier time than what has been seen before signed with the same key.)
```